CONTRIBUTING.md

```diff
@@ -19,6 +18,7 @@ In order to merge a Pull Request inside the `main` branch of the repository, the
  - Compiling code in Java 11
  - Passing test battery
  - Passing legacy compatibility check
+ - Passing Sonar quality gate
```

Comment:
I don't see sonar configuration in the CI

Reply:
It's done throught Github integration of the Sonar Analysis App. The quality gate is now a mandatory check

Reply:
I wonder why sonar configuration was not straightforward for pivot then :thinking:

Reply:
It's a small project (<10k LoC), by default, it is run automatically by the Sonar app on any existing push on an existing PR